docs/UserGuide.md

```diff
@@ -100,6 +100,7 @@ Column Name | Explanation
 Repository's Location | The `GitHub URL` or `Disk Path` to the git repository
 Branch | The branch to analyse in the target repository
 [Optional] Ignore Global List | The list of file path globs to ignore during analysis for each author. More details on the Java glob standard [here](https://javapapers.com/java/glob-with-java-nio/)
+[Optional] Ignore standalone config | Ignore the usage of standalone config in repository. To ignore the usage of standalone config in target repository, please enter **`yes`** in this column. Otherwise, the configuration in the target repository will be used by default.
```

Comment:
Ignore the ~usage~ `presence` of standalone config in `target` repository. ~To ignore the usage of standalone config in target repository~ `To do so`,